File: ExamSolution/Examshubham94/JavaExam/Question1/src/Date.java
package college;
public class Date{
	private String day;
	private String month;
	private int year;
	public Date(String d, String m, int y){
		day = d;
		month = m;
		year = y;
	
	}
	public Date(){}
	public String getDay(){return day;}
	public void setDay(String d)
	{day = d;}

	public String getMonth(){return month;}
	public void setMonth(String m)
	{month = m;}

	public int getYear(){return year;}
	public void setYear(int y)
	{year = y;}
	


}

File: ExamSolution/93 Shreyash/Dotnet/StudentDate/Date.cs

public class Date 
{
    public Date(string day , string month, string year)
    {
        Day = day;
        Month = month;
        Year = year;
        
    }
    public static string Day { get; set; }

    public static string Month {get; set;}

    public static string Year { get; set; }

    public static Date ParseDate (string dob)
    {
        string[] DateOfBirth = dob.Split('/');
        Day = DateOfBirth[0];
        Month = DateOfBirth[1];
        Year = DateOfBirth[2];
         var x = new Date(Day, Month, Year);
        return x;
    }
 
}